StructureController.cs

```diff
@@ -452,12 +453,20 @@ public string ToString(string selected)
     }
     public override string ToString()
     {
-        string output = "";
+        string output = $"{{\n\"position\": [{this.center_of_mass.x + this.transform.position.x}, {this.center_of_mass.y + this.transform.position.y}, {this.rotator.rotation.z}],\n\"translation\": [{this.translation.x}, {this.translation.y}],\n";
+        
+        output += "\"components\": [\n";
         foreach (var component in components.Values)
         {
-            output += component.ToString() + "\n";
+            output += component.ComponentToString() + "\n";
         }
-        return output;
+        output += "]\n\"entities\": [\n";
+
+        foreach (var entity in GameObject.Find("World").GetComponentsInChildren<ProjectileController>())
+        {
+            output += $"\"{entity.name}\": [{entity.transform.position.x}, {entity.transform.position.y},{entity.speed},{entity.transform.localEulerAngles.z.ToString("0.00")}]\n";
```

Comment:
Comma at the end of the line?

Reply:
],\n
 ^- since its a list

Reply:
![image](https://user-images.githubusercontent.com/20609480/191155868-df203140-d570-41a9-8b9a-208362540074.png)
It's parent is an array also

Reply:
I mean for entities, not components?